modules/mocked_authentication/spec/lib/credential/service_spec.rb

```diff
@@ -131,6 +131,13 @@
     let(:auto_uplevel) { false }
     let(:country) { 'USA' }
     let(:phone) { 'some-phone' }
+    let(:digest) { 'some-digest' }
```

Comment:
I see this updated mocked auth specs very slightly... Did we check and make sure mocked auth is still working after these changes?